lib/request.js

```diff
@@ -58,13 +64,21 @@ function buildRequest (opts) {
   }
 
   function handleHttp1Req (opts, done) {
+    if (opts.certificate) {
+      agents['clone'] = new https.Agent(agentOption({
+        ...opts, ...opts.certificate
+      }))
+      // if (Buffer.isBuffer(opts.certificate.key) && Buffer.isBuffer(opts.certificate.cert)) {
```

Comment:
Why is this commented out?

Reply:
Hello @jsumners , 

I was not sure if you want to check if `cert` and `key` are ` Buffer` or not, let me know if you want me to uncomment it.

Reply:
Hello @jsumners , let me know if you want me to leave the `buffer` test and if you want me to throw an exception.